### DISCUSSION TITLE
Stance regarding LightningCSS

### DISCUSSION THREAD

**Discussion by NatanLifshitz** [mantinedev/mantine#4795] [Feedback]
Vite [recently added support for LightningCSS](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#440-2023-07-06), which seems to have some benefits, but does not support a CSS preprocessor. 
At around the same time, Mantine started recommending using a CSS preprocessor (in 7.0.0). 

I'm wondering what is the current best practice regarding this and if there is any plan to address this incompatibility. Currently the options are:
-  Do not use [postcss-preset-mantine](https://mantine.dev/styles/postcss-preset), which is not recommended.
- Do not use LightningCSS, which has a performance impact both on the dev front and in the production front.

**Comment by rtivital** [comment]
You can use any tools for styling, it is not required to use mantine PostCSS preset – it is just a set of functions and mixins. You can write pure css, although it will be 1.5x – 2x of code that you will need to write if you need to handle both light and dark color schemes in your application.
